src/styles/components/_form.sass

```diff
@@ -71,3 +71,18 @@ form
       i
         margin-right: 8px
         color: #23c451
+
+.multi-select
+  input
+    border-color: #3a5f7b !important
```

Comment:
the library I used for this multiselect dropdown seems to add styles to the elements, so with the highest specificity possible... abusing `!important` seemed like the only way to override this.